src/commands/contributor/extension/publish.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { ExecRenderBaseCommand } from "../../../lib/basecommands/ExecRenderBaseCommand.js";
+import {
+  makeProcessRenderer,
+  processFlags,
+} from "../../../rendering/process/process_flags.js";
+import { Success } from "../../../rendering/react/components/Success.js";
+import {
+  extensionManifestArg,
+  parseExtensionManifest,
+} from "../../../lib/resources/extension/args_contributor.js";
+
+export default class Publish extends ExecRenderBaseCommand<
+  typeof Publish,
+  void
+> {
+  static description = "Publish an extension on the marketplace";
```

Comment:
The publishing of extensions will only work if verification has been requested and successfully completed beforehand.

I believe the corresponding route to request verification has not yet been published

Reply:
Can we rely on the API to respond with an appropriate and sensible error message in this case (in which case we could leave this as-is and just present the API error message), or should we catch that condition (successful verification) beforehand client-side?

Reply:
The API will likely respond with a reasonable error message. I just wanted to point out, the current flow will be interrupted by writing some mittwald guys to verify the extension (including starting the verification process) until the corresponding API route is published.